grid-cli/cmd/get_contracts.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"text/tabwriter"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-cli/internal/config"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/deployer"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/graphql"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/workloads"
+)
+
+// getContractsCmd represents the get contracts command
+var getContractsCmd = &cobra.Command{
+	Use:   "contracts",
+	Short: "Get twin contracts",
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg, err := config.GetUserConfig()
+		if err != nil {
+			log.Fatal().Err(err).Send()
+		}
+		t, err := deployer.NewTFPluginClient(cfg.Mnemonics, "sr25519", cfg.Network, "", "", "", 100, false)
+		if err != nil {
+			log.Fatal().Err(err).Send()
+		}
+		contracts, err := t.ContractsGetter.ListContractsByTwinID([]string{"Created"})
+		if err != nil {
+			log.Fatal().Err(err).Send()
+		}
+		printContractTables(contracts, cmd.OutOrStdout())
+
+	},
+}
+
+func init() {
+	getCmd.AddCommand(getContractsCmd)
+
+}
+
+func printContractTables(contracts graphql.Contracts, writer io.Writer) {
+	fmt.Println("Node contracts:")
```

Comment:
You have a writer, why would you do Printlns instead of Fprintlns?

Reply:
You're right I didn't notice that :)